README.md

```diff
@@ -28,6 +28,26 @@ JSON.json(j)
 #  "{\"an_array\":[\"string\",9],\"a_number\":5.0}"
 ```
 
+### Macro Strings
+
+`JSON` and `J` can be used to embed JSON data directly into source code:
```

Comment:
wouldn't just `json"""` be more Julian? Either way, :-1: to two ways to do same thing